src/System/Interactions.jl

```diff
@@ -199,11 +199,11 @@ function energy(sys::System{N}) where N
     for i in 1:natoms(crystal)
         if is_homogeneous(sys)
             ints = interactions_homog(sys)
-            E += energy_aux(sys, ints[i], i, all_cells(sys))
+            E += energy_aux(sys, ints[i], i, all_cells(sys), homog_bond_iterator(latsize))
         else
-            ints = interactions_inhomog(sys)
             for cell in all_cells(sys)
-                E += energy_aux(sys, ints[cell, i], i, (cell, ))
+                ints = interactions_inhomog(sys)
```

Comment:
Not sure why this is in the loop now, but shouldn't have a bad performance impact either way